### ISSUE TITLE
a11y: Share Buttons Accessibility using keyboard Tab navigation

### ISSUE THREAD

**Issue by ramiy** [elementor/elementor#11299]
_(original ticket opened in gitlab a month ago)_

**The Problem**

The "*Share Buttons Widget*" (Elementor Pro) is not fully accessible.

**What do we support?**

1. Share buttons have `aria` tags.
1. Share buttons also have labels for screen readers.

**What is missing?**

Seems like disabled people navigating using `Tab` key on their keyboard can't "see" the buttons. The buttons are not included in the tab-index.

Why? because the wrapper element is a `<div>` ([see here](https://gitlab.com/elementor/elementor-pro/-/blob/develop/modules/share-buttons/widgets/share-buttons.php#L611)), div elements have no **focus** on them when you navigate using your keyboard. Share buttons should use a wrapper element that can be focused.

**Solution**

Using `tabindex="0"` attribute to the Share Button has a **focus** on the container `div` when the user navigates using  a keyboard.

But this is a temporary solution, the wrapper element should be a `<button>` or an `<a>` - [like the social icons widget](https://github.com/elementor/elementor/blob/master/includes/widgets/social-icons.php#L572-L580).

**Comment by ramiy** [comment]
How to fix the issue - from the original PR on gitlab:

![999](https://user-images.githubusercontent.com/576623/81078515-075fb300-8ef7-11ea-990d-68ed02972083.png)

**Comment by ramiy** [comment]
@joshmarom @matipojo

**Comment by Bovelett** [comment]
A +1 from me. :)

**Comment by rami-elementor** [comment]
Ok, this issue has been resolved. The next Elementor Pro patch will include a fix.

Thanks!

**Comment by Bovelett** [comment]
Thanks!! ❤️
